src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/36.diff.sql

```diff
@@ -0,0 +1,748 @@
+SET XACT_ABORT ON
+
+IF NOT EXISTS
+(
+    SELECT *
+    FROM sys.all_columns c
+    INNER JOIN sys.tables t ON t.object_id = c.object_id
+    INNER JOIN sys.default_constraints d ON c.default_object_id = d.object_id
+    WHERE t.name = 'ChangeFeed' AND c.name = 'Timestamp'
+)
+BEGIN
+
+    ALTER TABLE dbo.ChangeFeed ADD DEFAULT SYSDATETIMEOFFSET() FOR Timestamp
+
+END
+GO
+
+DROP INDEX IF EXISTS IXC_ChangeFeed ON dbo.ChangeFeed
+DROP INDEX IF EXISTS IX_ChangeFeed_Sequence ON dbo.ChangeFeed
+
+CREATE UNIQUE CLUSTERED INDEX IXC_ChangeFeed ON dbo.ChangeFeed
+(
+    Timestamp,
+    Sequence
+)
```

Comment:
You can do with(ONLINE=ON and DROP_EXISTING=TRUE)

There should be samples in other diff files

Reply:
I think, it doesn't work for clustered index. I may be wrong